### PR TITLE
Fixes the pipe dispenser dispending random pipes.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
title

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/game/machinery/pipe/construction.dm [hunk 1]
```diff
@@ -13,44 +13,44 @@ Buildable meters
...
 #define PIPE_SCRUBBER			10
-#define PIPE_GAS_FILTER			13
-#define PIPE_GAS_MIXER			14
-#define PIPE_PASSIVE_GATE       15
-#define PIPE_VOLUME_PUMP        16
-#define PIPE_HEAT_EXCHANGE      17
-#define PIPE_MTVALVE			18
-#define PIPE_MANIFOLD4W			19
-#define PIPE_CAP				20
+#define PIPE_GAS_FILTER			11
```
Comment (SleepyGemmy) [inline code/game/machinery/pipe/construction.dm:16]: not in use anymore.

File: code/game/machinery/pipe/construction.dm [hunk 2]
```diff
@@ -13,44 +13,44 @@ Buildable meters
...
+#define PIPE_GAS_MIXER			12
```
Comment (SleepyGemmy) [inline code/game/machinery/pipe/construction.dm:17]: not in use anymore.

File: code/game/machinery/pipe/construction.dm [hunk 3]
```diff
@@ -13,44 +13,44 @@ Buildable meters
...
-#define PIPE_DOWN				22
+#define PIPE_UP					19
+#define PIPE_DOWN				20
 ///// Z-Level stuff
-#define PIPE_GAS_FILTER_M		23
-#define PIPE_GAS_MIXER_T		24
-#define PIPE_GAS_MIXER_M		25
-#define PIPE_OMNI_MIXER			26
-#define PIPE_OMNI_FILTER		27
+#define PIPE_GAS_FILTER_M		21
```
Comment (SleepyGemmy) [inline code/game/machinery/pipe/construction.dm:28]: not in use anymore.

File: code/game/machinery/pipe/construction.dm [hunk 4]
```diff
@@ -13,44 +13,44 @@ Buildable meters
...
+#define PIPE_GAS_MIXER_T		22
```
Comment (SleepyGemmy) [inline code/game/machinery/pipe/construction.dm:29]: not in use anymore.

File: code/game/machinery/pipe/construction.dm [hunk 5]
```diff
@@ -13,44 +13,44 @@ Buildable meters
...
+#define PIPE_GAS_MIXER_M		23
```
Comment (SleepyGemmy) [inline code/game/machinery/pipe/construction.dm:30]: not in use anymore.

**Review by SleepyGemmy — COMMENTED** [review]
deleted

**Pull request merged**